web/src/app/admin/admin-routing.module.ts

```diff
@@ -0,0 +1,21 @@
+// Core modules
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+// DashboardHub components
+import { AdminGuard } from '@app/core/guards/admin.guard';
+import { AdminComponent } from './admin.component';
+
+const routes: Routes = [
+  {
+    path: 'users',
+    component: AdminComponent,
```

Comment:
If the module is called `admin`, we should be more specific with the `component` name, especially as there will be more components/pages in the admin module later on

Reply:
okay will create users-list component and move everything there